File: Interpreter C#/Interpreter/MyList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Interpreter
{
    namespace Model
    {
        class MyList<T> : MyIList<T>
        {
            List<T> list;
            public MyList()
            {
                this.list = new List<T>();
            }
            void MyIList<T>.add(T v)
            {
                this.list.Add(v);
            }

            void MyIList<T>.remove(T v)
            {
                this.list.Remove(v);
            }

            public override string ToString()
            {
                string str = " ";
                foreach(T i in this.list)
                {
                    str = str + i.ToString();
                }
                return str;
            }
        }
    }

}

File: MAP/Interpretor2/src/Model/MyList.java
package Model;

import java.util.*;

public class MyList<T> implements MyIList<T> {

	ArrayList<T> list;
	public MyList(){
		this.list=new ArrayList<T>();
	}
	public void add(T i)
	{
		this.list.add(i);
	}
	public void remove(T i)
	{
		this.list.remove(i);
	}
	@Override
	public String toString(){
		String str=" ";
		for(T t:this.list)
			str=str+" "+t;
		return str;
	}
	
}
